Makefile

```diff
@@ -32,12 +32,14 @@ SUB := $(addsuffix .sub, $(ALL))
 build: build.sub
 build-linux: build-linux.sub
 build-darwin: build-darwin.sub
-clean: clean.sub
 docker: docker.sub
 lint: lint.sub
 push: push.sub
 test: test.sub
 
+clean: clean.sub
+	rm -rf bin
```

Comment:
"$(ROOT)/bin"

I'd probably use that over just 'bin' ... same thing however, so up to you.